Examples/zib-MedicationAdministration-02.xml

```diff
@@ -0,0 +1,38 @@
+<MedicationAdministration xmlns="http://hl7.org/fhir" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://hl7.org/fhir http://hl7.org/fhir/STU3/medicationadministration.xsd">
+    <id value="zib-medicationadministration-02"/>
+    <meta>
+        <profile value="http://nictiz.nl/fhir/StructureDefinition/zib-MedicationAdministration"/>
```

Comment:
Misschien hier en disclaimer in comment geven dat dit een zeer minimalistisch en waarschijnlijk geen realitische MedicationAdministration is?

Reply:
Done